seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/config/BaseFileSinkConfig.java

```diff
@@ -38,6 +38,8 @@
 public class BaseFileSinkConfig implements DelimiterConfig, Serializable {
     private static final long serialVersionUID = 1L;
     protected CompressFormat compressFormat = BaseSinkConfig.COMPRESS_CODEC.defaultValue();
+    protected ArchiveCompressFormat archiveCompressFormat =
```

Comment:
does sink support this feature?

Reply:
No support yet， labeled todo in `baseSinkConfig`

Reply:
Please remove it if useless at now.

Reply:
get

Reply:
Sink config is deleted. only todo is retained